contrib/guix/libexec/build.sh

```diff
@@ -452,11 +452,7 @@ mv --no-target-directory "$OUTDIR" "$ACTUAL_OUTDIR" \
 
 (
     cd /outdir-base
-    {
-        echo "$GIT_ARCHIVE"
-        find "$ACTUAL_OUTDIR" -type f
-    } | xargs realpath --relative-base="$PWD" \
-      | xargs sha256sum \
+    find "$GIT_ARCHIVE" "$ACTUAL_OUTDIR" -type f -execdir bash -c 'basename "$1" | xargs sha256sum' _ {} \; \
```

Comment:
To make find print the filename without path, you can simply use `find -printf "%f\n"`, this would result in:
`find "$GIT_ARCHIVE" "$ACTUAL_OUTDIR" -type f -printf "%f\n" | xargs sha256sum' _ {}`